farmdata2_modules/fd2_tabs/fd2_field_kit/seedingInput.html

```diff
@@ -83,8 +101,36 @@ <h1 style='text-align:center;'>Seeding Input Log</h1>
                 areaToIDMap: new Map(),
                 logTypeToIDMap: new Map(),
                 plantingId: 0,
+                alertVal: false,
             },
-            methods:{
+            methods:{    
+                stopAlert(){
+                    this.alertVal = false
+                },
+                clearForm(){
+                    this.cropArray.push('')
+                    this.areaFilter.push('')
+                    console.log(this.cropArray)
+                    console.log(this.areaFilter)
+                    this.selectedDate = dayjs().format('YYYY-MM-DD').toString()
+                    this.selectedCrop = "ARUGULA"
```

Comment:
Should the selectedCrop be set to '' here in the same way the selectedArea is below?